source/portable/NetworkInterface/STM32Fxx/NetworkInterface.c

```diff
@@ -628,7 +600,7 @@ BaseType_t xSTM32F_NetworkInterfaceInitialise( NetworkInterface_t * pxInterface
     }
     else
     {
-        if( xPhyObject.ulLinkStatusMask != 0U )
+        if( xSTM32F_GetPhyLinkStatus( pxInterface ) != pdFALSE )
```

Comment:
Can you please revert this change?
The driver is testing the field `ulLinkStatusMask` which is being maintained by the EMAC task.